cmd/utils/selector.go

```diff
@@ -1,514 +0,0 @@
-// Copyright 2022 The Okteto Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package utils
-
-import (
-	"bytes"
-	"context"
-	"errors"
-	"fmt"
-	"io"
-	"os"
-	"runtime"
-	"strings"
-	"text/template"
-
-	"github.com/chzyer/readline"
-	"github.com/juju/ansiterm"
-	"github.com/manifoldco/promptui"
-	"github.com/manifoldco/promptui/list"
-	"github.com/manifoldco/promptui/screenbuf"
-	oktetoLog "github.com/okteto/okteto/pkg/log"
-	"github.com/okteto/okteto/pkg/okteto"
-	"golang.org/x/term"
-)
-
-const (
-	esc        = "\033["
-	showCursor = esc + "?25h"
-)
-
-// ErrInvalidOption is raised when the selector has an invalid option
-var ErrInvalidOption = errors.New("invalid option")
-
-// OktetoSelector represents the selector
-type OktetoSelector struct {
-	Label string
-	Items []SelectorItem
-	Size  int
-
-	Templates *promptui.SelectTemplates
-	Keys      *promptui.SelectKeys
-
-	OktetoTemplates *oktetoTemplates
-}
-
-//oktetoTemplates stores the templates to render the text
-type oktetoTemplates struct {
-	FuncMap   template.FuncMap
-	label     *template.Template
-	active    *template.Template
-	inactive  *template.Template
-	selected  *template.Template
-	details   *template.Template
-	help      *template.Template
-	extraInfo *template.Template
-}
-
-//SelectorItem represents a selectable item on a selector
-type SelectorItem struct {
-	Name      string
-	Label     string
-	Enable    bool
-	IsOkteto  bool
-	Namespace string
-	Builder   string
-	Registry  string
-}
-
-//AskForOptionsOkteto given some options ask the user to select one
-func AskForOptionsOkteto(ctx context.Context, options []SelectorItem, label, selectedTpl string) (string, bool, error) {
-	selectedTemplate := getSelectedTemplate(selectedTpl)
-	activeTemplate := getActiveTemplate()
-	inactiveTemplate := getInactiveTemplate()
-
-	prompt := OktetoSelector{
-		Label: label,
-		Items: options,
-		Size:  len(options),
-		Templates: &promptui.SelectTemplates{
-			Label:    "{{ .Label }}",
-			Selected: selectedTemplate,
-			Active:   activeTemplate,
-			Inactive: inactiveTemplate,
-			FuncMap:  promptui.FuncMap,
-		},
-	}
-
-	prompt.Templates.FuncMap["oktetoblue"] = oktetoLog.BlueString
```

Comment:
new funcion Ask() is now performing the question, using the pointer of the OktetoSelector with the info regarding the selector